pkg/apis/app/v2/kieapp_types.go

```diff
@@ -152,18 +152,31 @@ type KieServerSet struct {
 	KafkaJbpmEventEmitters *KafkaJBPMEventEmittersObject `json:"kafkaJbpmEventEmitters,omitempty"`
 	Cors                   *CORSFiltersObject            `json:"cors,omitempty"`
 	// MDBMaxSession number of KIE Executor sessions
-	MDBMaxSession *int `json:"MDBMaxSession,omitempty"`
+	MDBMaxSession    *int              `json:"MDBMaxSession,omitempty"`
+	TerminationRoute *TerminationRoute `json:"terminationRoute,omitempty"`
+}
+
+type TerminationRoute struct {
+	// +kubebuilder:validation:Enum:=http;https
+	// Protocol to use, this couldmoverride the global disableSSL, http is the default, otherwise https
+	Protocol string `json:"protocol,omitempty"`
+	// +kubebuilder:validation:Enum:=passthrough;edge
+	// Passthrough is the default protocol, otherwise edge could be used.
```

Comment:
maybe a better description about edge configuration telling that it only supports http?
And what happen if user select edge with https protocol?

Reply:
ok about the description, edge means http but we can't enforce in the ui, we could improve the description